executor/runtime/docker/docker.go

```diff
@@ -2140,14 +2140,20 @@ func (r *DockerRuntime) k8sContainerToDockerConfigs(c *runtimeTypes.ExtraContain
 	if err != nil {
 		return nil, nil, nil, fmt.Errorf("Parsing the liveness probe for the %s container: %w", v1Container.Name, err)
 	}
+	entrypoint := computeExtraContainersDockerEntrypoint(c)
+	command := computeExtraContainersDockerCommand(c)
+	if len(entrypoint) == 0 && len(command) == 0 {
+		err := fmt.Errorf("An Entrypoint or Command must be specified for the container")
```

Comment:
nit: why not just return `fmt.Errorf(...)` directly?

Reply:
eh, just shorter lines